src/generate-changelog.ts

```diff
@@ -185,80 +185,51 @@ async function getReleases(graphqlWithAuth: Function): Promise<ReleaseInfo[]> {
   const releases = response.repository.releases.nodes as GraphQLRelease[]
 
   for (const release of releases) {
-    releaseInfo.push({name: release.name, version: release.tag.name})
+    releaseInfo.push({ name: release.name, version: release.tag.name })
   }
 
   return releaseInfo
 }
 
+// Given a description and a regex, parses the description, looking for all lines that match
+// the regex. Any matches that aren't the default template text will be added to `changelogSectionArray`.
+function parseMatchesFromDescription(prDescription: string, regex: RegExp, changelogSectionArray: string[]): void {
```

Comment:
should this just return an array of entries instead? From an API perspective, it seems odd that we are passing in an array that we internally mutate instead of just returning the list of matches directly.

Reply:
Fair point! Will switch.